<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLineEditor</class>
 <widget class="QWidget" name="CLineEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>436</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="2">
        <widget class="QLineEdit" name="lePassword">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="btnConnect">
         <property name="text">
          <string>connect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="leServerAddress"/>
       </item>
       <item row="6" column="3">
        <widget class="QPushButton" name="btnLoad">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>load</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblUser">
         <property name="text">
          <string>FTP user</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="leFilePath">
         <property name="text">
          <string>/etc/CCcam.cfg</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lblFilePath">
         <property name="text">
          <string>Remote CCcam.cfg path</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lblServerAddress">
         <property name="text">
          <string>Server address</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblPassword">
         <property name="text">
          <string>FTP password</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="leUsername"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblServerPort">
         <property name="text">
          <string>Server port</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="sbServerPort">
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>65535</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="btnAdd">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnRemove">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>remove</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btnClear">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>clear</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="6">
        <widget class="QTableView" name="twCLines">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="btnSave">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leServerAddress</tabstop>
  <tabstop>leUsername</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>leFilePath</tabstop>
  <tabstop>twCLines</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnSave</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnAdd</sender>
   <signal>clicked()</signal>
   <receiver>CLineEditor</receiver>
   <slot>addCLine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>411</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>429</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemove</sender>
   <signal>clicked()</signal>
   <receiver>CLineEditor</receiver>
   <slot>removeCLine()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSave</sender>
   <signal>clicked()</signal>
   <receiver>CLineEditor</receiver>
   <slot>saveCLines()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>886</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>934</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLoad</sender>
   <signal>clicked()</signal>
   <receiver>CLineEditor</receiver>
   <slot>loadCLines()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>930</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>936</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnConnect</sender>
   <signal>clicked()</signal>
   <receiver>CLineEditor</receiver>
   <slot>connectToFTP()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>930</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>935</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClear</sender>
   <signal>clicked()</signal>
   <receiver>CLineEditor</receiver>
   <slot>clearCLines()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePassword</sender>
   <signal>returnPressed()</signal>
   <receiver>CLineEditor</receiver>
   <slot>connectToFTP()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>753</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leFilePath</sender>
   <signal>returnPressed()</signal>
   <receiver>CLineEditor</receiver>
   <slot>loadCLines()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>419</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addCLine()</slot>
  <slot>removeCLine()</slot>
  <slot>saveCLines()</slot>
  <slot>loadCLines()</slot>
  <slot>connectToFTP()</slot>
  <slot>clearCLines()</slot>
 </slots>
</ui>
